server/routes/dpr/index.ts

```diff
@@ -24,12 +25,18 @@ export function dprRouter(): Router {
             apiUrl: config.apis.hmppsIncidentReportingApi.url,
             apiTimeout: config.apis.hmppsIncidentReportingApi.timeout.deadline,
             layoutTemplate: 'partials/dprLayout.njk',
-            tokenProvider: defaultTokenProvider,
+            tokenProvider: (req: Request, res: Response, next: NextFunction) => {
+              return res.locals.systemToken
+            },
           }),
         )
       }
     }
+  })
 
+  router.get('/', async (req, res) => {
+    const { incidentReportingApi } = res.locals.apis
+    const definitions = await incidentReportingApi.getManagementReportDefinitions()
```

Comment:
We get the definitions again because we want DPR to apply the policies for THIS logged in user to know if the report is accessible